.github/workflows/merge_db.yml

```
@@ -0,0 +1,20 @@
+name: Merge Databases
+on:
+  schedule:
+    - cron: "*/10 * * * *" # Every 15 mins
+  workflow_dispatch:
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)